HACKING.md

```diff
@@ -367,9 +387,15 @@ Common instructions for porting:
     };
     ```
     Where `convert_to_json` is a function that converts `$self->prop_name` to
-    its appropriate JSON representation. If `prop_name` is an entity that has
-    a `TO_JSON` method defined, it will be called automatically and you can
-    simply do `prop_name => $self->prop_name`.
+    its appropriate JSON representation.
+    
+    If `prop_name` is an entity that has a `TO_JSON` method defined, it will be
+    called automatically and you can simply do `prop_name => $self->prop_name`.
```

Comment:
This is no longer true since https://github.com/metabrainz/musicbrainz-server/pull/1958 -- you have to call `prop_name => $self->prop_name->TO_JSON` (or `prop_name => to_json_object($self->prop_name)` if you want to handle undef values).